### ISSUE TITLE
how to add val loss

### ISSUE THREAD

**Issue by yang-0201** [meituan/YOLOv6#649]
### Search before asking

- [X] I have searched the YOLOv6 [issues](https://github.com/meituan/YOLOv6/issues) and found no similar feature requests.

### Description

In the training process, val loss is added to facilitate the judgment of whether the network is over fitted

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by Chilicyy** [comment]
Thanks for your attention! We have received your suggestion. Also, you can refer to the training part, and add the val loss. Welcome to pull requests.

**Comment by yang-0201** [comment]
谢谢，我尝试加入val loss，但是我尝试之后遇到了一些问题，在训练过程中val loss不会下降，反而慢慢上升。
我的实现方式如下：
首先在engine.py中的验证阶段eval_model（）函数中传入之前构建的self.compute_loss和train_epoch参数，然后在eval.py 中val.predict_model(）也传入compute_loss，然后在验证阶段计算val loss时把model forward返回值设置为和训练时候一样：return x, cls_score_list, reg_distri_list，在evaler.py 中128行的循环中加入：
outputs, _ = model(imgs,val_loss = True)
targets = targets.to(self.device)
total_loss, loss_items = compute_loss(outputs, targets, train_epoch, 15)
print(total_loss,loss_items)
outputs, _ = model(imgs, val_loss=False)
然而训练过程中loss在下降，显示的val loss却不下降，请教一下是什么问题！

**Comment by mtjhl** [comment]
val 的精度在正常上升吗？

**Comment by yang-0201** [comment]
是的，精度在上升，训练的loss也在正常下降
